src/apps/intel_mp/intel_mp.lua

```diff
@@ -324,7 +326,9 @@ function Intel:new (conf)
       vlan = conf.vlan,
       want_mirror = conf.mirror,
       rxcounter = conf.rxcounter,
-      txcounter = conf.txcounter
+      txcounter = conf.txcounter,
+      limit = conf.rate_limit,
```

Comment:
I'd rather keep calling this `rate_limit`, because just `limit` is unclear. What do you think?